charts/create-secret-manager-secret/ci/test-values.yaml

```diff
@@ -1,8 +0,0 @@
----
```

Comment:
Per @tcjiang9 , these `ci/*.yaml` are removed since the `test-values.yaml` in the datarepo umbrella chart is used instead.